clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/config/Justification.java

```diff
@@ -26,20 +27,24 @@ public class Justification {
   private static final String JUSTIFICATION_ENABLED_ENV_KEY = "AUDIT_CLIENT_JUSTIFICATION_ENABLED";
 
   @JsonProperty("public_keys_endpoint")
-  private String publicKeysEndpoint = "localhost:8080";
+  private String publicKeysEndpoint;
 
   @JsonProperty("enabled")
   private boolean enabled;
 
   public String getPublicKeysEndpoint() {
-    return System.getenv()
-        .getOrDefault(JUSTIFICATION_PUBLIC_KEYS_ENDPOINT_ENV_KEY, publicKeysEndpoint);
+    return ConfigUtils.getEnvOrDefault(
```

Comment:
For future, let's separate such refactoring to be its own PR. They are not essential to the change at hand and cause distraction for reviewers.